target/linux/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -85,6 +85,11 @@ platform_do_upgrade() {
 		CI_ROOT_UBIPART="rootfs"
 		nand_do_upgrade "$1"
 		;;
+	netgear,rax120v2)
+		CI_KERN_UBIPART="kernel"
```

Comment:
You sure that this works?

In my testing using `kernel` as the kernel partition name would trigger the old raw nandwrite instead of UBI flashing

Reply:
Also alphabetical ordering again

Reply:
It seems to work. The following is displayed in dmesg after flashing. 

[    7.844917] UBIFS (ubi0:2): Mounting in unauthenticated mode
[    7.845021] UBIFS (ubi0:2): background thread "ubifs_bgt0_2" started, PID 760
[    7.927255] UBIFS (ubi0:2): UBIFS: mounted UBI device 0, volume 2, name "rootfs_data"
[    7.927303] UBIFS (ubi0:2): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[    7.934101] UBIFS (ubi0:2): FS size: 259031040 bytes (247 MiB, 2040 LEBs), max 2053 LEBs, journal size 12951552 bytes (12 MiB, 102 LEBs)
[    7.943997] UBIFS (ubi0:2): reserved for root: 4952683 bytes (4836 KiB)
[    7.956307] UBIFS (ubi0:2): media format: w5/r0 (latest is w5/r0), UUID 73F1A91B-1765-4BE4-B4C6-1FE79C28A0C8, small LPT model
[    7.963600] mount_root: overlay filesystem has not been fully initialized yet
[    7.974433] mount_root: switching to ubifs overlay

Reply:
Can you capture the sysupgrade log?

Reply:
`Fri Feb 24 11:06:47 UTC 2023 upgrade: Commencing upgrade. Closing all shell sessions.
Command failed: Connection failed
Watchdog handover: fd=3
- watchdog -
Watchdog did not previously reset the system
Fri Feb 24 11:06:48 UTC 2023 upgrade: Sending TERM to remaining processes ...
Fri Feb 24 11:06:48 UTC 2023 upgrade: Sending signal TERM to hostapd (1985)
Fri Feb 24 11:06:48 UTC 2023 upgrade: Sending signal TERM to ntpd (2676)
Fri Feb 24 11:06:48 UTC 2023 upgrade: Sending signal TERM to ntpd (2682)
Fri Feb 24 11:06:52 UTC 2023 upgrade: Sending KILL to remaining processes ...
Fri Feb 24 11:06:52 UTC 2023 upgrade: Sending signal KILL to ntpd (2676)
Fri Feb 24 11:06:52 UTC 2023 upgrade: Sending signal KILL to ntpd (2682)
[   56.092947] stage2 (3326): drop_caches: 3
Fri Feb 24 11:06:58 UTC 2023 upgrade: Switching to ramdisk...
Fri Feb 24 11:06:59 UTC 2023 upgrade: Performing system upgrade...
verifying sysupgrade tar file integrity
Unlocking kernel ...

Writing from <stdin> to kernel ...
[   57.254555] ubi1: attaching mtd24
[   57.382503] ubi1: scanning is finished
[   57.388123] ubi1: attached mtd24 (name "kernel", size 29 MiB)
[   57.388158] ubi1: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[   57.392870] ubi1: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[   57.399621] ubi1: VID header offset: 2048 (aligned 2048), data offset: 4096
[   57.406487] ubi1: good PEBs: 232, bad PEBs: 0, corrupted PEBs: 0
[   57.413255] ubi1: user volume: 0, internal volumes: 1, max. volumes count: 128
[   57.419502] ubi1: max/mean erase counter: 1/0, WL threshold: 4096, image sequence number: 1290124076
[   57.426549] ubi1: available PEBs: 148, total reserved PEBs: 84, PEBs reserved for bad PEB handling: 80
[   57.435837] ubi1: background thread "ubi_bgt1d" started, PID 4046
UBI device number 1, total 232 LEBs (29458432 bytes, 28.0 MiB), available 148 LEBs (18792448 bytes, 17.9 MiB), LEB size 126976 bytes (124.0 KiB)
umount: can't unmount /dev/ubiblock0_1: Invalid argument
[   57.561620] block ubiblock0_1: released
Volume ID 1, size 50 LEBs (6348800 bytes, 6.0 MiB), LEB size 126976 bytes (124.0 KiB), dynamic, name "rootfs", alignment 1
Set volume size to 262205440
Volume ID 2, size 2065 LEBs (262205440 bytes, 250.0 MiB), LEB size 126976 bytes (124.0 KiB), dynamic, name "rootfs_data", alignment 1
Unlocking kernel ...

Writing from <stdin> to kernel ...
sysupgrade successful
umount: can't unmount /dev: Resource busy
umount: can't unmount /tmp: Resource busy
[   60.991573] leds white:5g: led_trigger_set: Error sending uevent
[   61.141535] leds white:24g: led_trigger_set: Error sending uevent
[   61.339737] remoteproc remoteproc0: stopped remote processor cd00000.q6v5_wcss
[   61.911672] reboot: Restarting system`